entrypoint.sh

```diff
@@ -32,7 +35,7 @@ fi
 
 echo {} >${INPUT_SARIF_FILE}
 
-tfsec --format=sarif "${INPUT_WORKING_DIRECTORY}" ${CONFIG_FILE_OPTION} ${TFVARS_OPTION} ${TFSEC_ARGS_OPTION} >${INPUT_SARIF_FILE}
+tfsec --soft-fail --force-all-dirs --format=sarif "${INPUT_WORKING_DIRECTORY}" ${CONFIG_FILE_OPTION} ${TFVARS_OPTION} ${TFSEC_ARGS_OPTION} >${INPUT_SARIF_FILE}
```

Comment:
Can we get away without `--force-all-dirs`? 

```suggestion
tfsec --soft-fail --format=sarif "${INPUT_WORKING_DIRECTORY}" ${CONFIG_FILE_OPTION} ${TFVARS_OPTION} ${TFSEC_ARGS_OPTION} >${INPUT_SARIF_FILE}
```